### DISCUSSION TITLE
Database Idea

### DISCUSSION THREAD

**Discussion by MaeghanColbyCorrea** [ccci-code/summerproject#15] [Ideas]
What if we used JavaScript to add information to and pull information from a form? So, on one page we could choose to what location we could add data to and then it would direct us to a HTML page in which we could add data to the form. And then, when we choose the meals, we could have JavaScript that automatically pulls the data from the form into the dropdown. That way, we don't have to continuously edit the JSON file when adding meal ideas.

**Comment by ccci-code** [comment]
Sounds great!  Do you have any ideas on how the form should appear?  I think having a separate page to enter the meals is a good idea.

**Reply by MaeghanColbyCorrea** [reply]
Yes, definitely. When adding items to the form, one webpage can have a dropdown where you choose what you want to add, and you may then add the item and what ingredients would be needed for it. When it comes to choosing items to randomly generate meal ideas, I feel like each the main, side, and vegetable should have a button where it redirects you to separate forms where you can choose the item from the form, which should be automatically alphabetized when an item is added instead of having a mile-long dropdown to scroll through. This would make it easier to find what the individual is looking for and it would be easier to program.
